### PR TITLE
OCM-7478 | ci: Automate OCP-73154,65928,66071

### PR DESCRIPTION
**Pull request by yuwang-RH**
new automated 73154 65928 66071
Logs see here https://privatebin.corp.redhat.com/?ede15f17cbdb1725#121xfBpCDaKGFLrA4ycXkiweGgFHtvM2fzwFCPqomM6

### CODE REVIEW

**Comment by yuwang-RH** [comment]
presubmit

**Comment by yuwang-RH** [comment]
@xueli181114 Could you help review?

File: tests/e2e/cluster_creation_test.go [hunk 1]
```diff
@@ -4,7 +4,11 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	CI "github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
```
Comment (xueli181114) [inline tests/e2e/cluster_creation_test.go:6]: This import should be removed. It is dup with the new import

File: tests/e2e/cluster_creation_test.go [hunk 2]
```diff
@@ -4,7 +4,11 @@ import (
...
+	ci "github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
+	cms "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/cms"
 	CON "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
```
Comment (xueli181114) [inline tests/e2e/cluster_creation_test.go:10]: This is dup import

File: tests/e2e/cluster_creation_test.go [hunk 3]
```diff
@@ -25,4 +29,29 @@ var _ = Describe("Create cluster", func() {
...
+
+			By("Create cluster again")
+			clusterArgs := &exec.ClusterCreationArgs{}
+			clusterService, err := EXE.NewClusterService(profile.GetClusterManifestsDir())
+			Expect(err).ToNot(HaveOccurred())
+			err = clusterService.Apply(clusterArgs, false, false)
+			Expect(err).ToNot(HaveOccurred())
+			clusterOutput, err := clusterService.Output()
+			Expect(err).ToNot(HaveOccurred())
+			Expect(clusterOutput.ClusterID).ToNot(BeEmpty())
```
Comment (xueli181114) [inline tests/e2e/cluster_creation_test.go:55]: Better to verify the new clusterID is different with previous one.

File: tests/e2e/cluster_creation_test.go (hunk 1 above)
Comment (yuwang-RH) [reply tests/e2e/cluster_creation_test.go:6]: updated.

File: tests/utils/cms/cms.go [hunk 4]
```diff
@@ -350,3 +353,43 @@ func ListNodePools(connection *client.Connection, clusterID string, parameter ..
...
+
+// Wait for cluster deleted via OCM
+func WaitClusterDeleted(connection *client.Connection, clusterID string, timeoutMinute ...int) error {
+	timeout := 30 * time.Minute
+	if len(timeoutMinute) == 1 {
+		timeout = time.Duration(timeoutMinute[0]) * time.Minute
+	}
+	start := time.Now()
+	for time.Since(start) < timeout {
+		fmt.Printf("Waiting for the cluster %s deleted. Timeout after %d mins\n",
```
Comment (xueli181114) [inline tests/utils/cms/cms.go:376]: Don't use fmt.Print. Use log instead

File: tests/utils/cms/cms.go [hunk 5]
```diff
@@ -350,3 +353,43 @@ func ListNodePools(connection *client.Connection, clusterID string, parameter ..
...
+			clusterID, int(math.Ceil(timeout.Minutes()-time.Since(start).Minutes())))
+		resp, _ := RetrieveClusterDetail(connection, clusterID)
+
+		if resp.Status() != CON.HTTPOK && resp.Status() != CON.HTTPNotFound {
+			err := fmt.Errorf(">>> [Error] Getting the cluster information meets error: %s", resp.Error().Reason())
+			return err
+		}
+
+		if resp.Status() == CON.HTTPNotFound {
+			fmt.Printf("OOH! The cluster  %s is deleted.\n", clusterID)
```
Comment (xueli181114) [inline tests/utils/cms/cms.go:386]: Use log instead of fmt.Print

File: tests/utils/cms/cms.go [hunk 6]
```diff
@@ -350,3 +354,43 @@ func ListNodePools(connection *client.Connection, clusterID string, parameter ..
...
+			clusterID, int(math.Ceil(timeout.Minutes()-time.Since(start).Minutes())))
+		resp, _ := RetrieveClusterDetail(connection, clusterID)
+
+		if resp.Status() != CON.HTTPOK && resp.Status() != CON.HTTPNotFound {
+			err := fmt.Errorf(">>> [Error] Getting the cluster information meets error: %s", resp.Error().Reason())
+			return err
+		}
+
+		if resp.Status() == CON.HTTPNotFound {
+			log.Printf("OOH! The cluster  %s is deleted.\n", clusterID)
```
Comment (xueli181114) [inline tests/utils/cms/cms.go:387]: Not this `log`
But [this one](https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/tests/utils/log/logger.go)

File: tests/e2e/cluster_creation_test.go [hunk 7]
```diff
@@ -4,7 +4,9 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	CI "github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
+	cms "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/cms"
```
Comment (radtriste) [inline tests/e2e/cluster_creation_test.go:7]: `cms` is not needed as it is the name of the package

File: tests/e2e/cluster_creation_test.go [hunk 8]
```diff
@@ -25,4 +27,31 @@ var _ = Describe("Create cluster", func() {
 				Expect(err).ToNot(HaveOccurred())
 			}
 		})
+
+	It("Cluster can be recreated if it was deleted not from tf - [id:66071]", CI.Day3, CI.Medium,
```
Comment (radtriste) [inline tests/e2e/cluster_creation_test.go:31]:
```suggestion
	It("Cluster can be recreated if it was not deleted from tf - [id:66071]", CI.Day3, CI.Medium,
```

**Review by radtriste — APPROVED** [review]
minor changes

**Comment by yuwang-RH** [comment]
@radtriste I did update with your comments. could you help review again? Thanks.

**Pull request merged**